internal/netdev/netdev.go

```diff
@@ -188,7 +206,7 @@ func getInterfaceState(data *IfaceData) error {
 // Get interfaces statistics
 // @result: ifaceStats, err
 func getInfacesStatistics(data *IfaceData) error {
-	basePath := "/sys/class/net/" + data.Name + "/statistics"
+	basePath := netDevicePath + data.Name + "/statistics"
```

Comment:
Maybe we can use https://pkg.go.dev/path#Join

Reply:
done